### ISSUE TITLE
Installing Net-Whois-RIPE-2.006002 not succesful

### ISSUE THREAD

**Issue by netravnen** [rfc1036/rpsltool#2]
Had to open an [issue with Net::Whois::RIPE](https://github.com/arhuman/Net-Whois-RIPE/issues/23) for hopefully getting Net::Whois::RIPE to be able to install. Looks like the module has an issue regarding an test it do.

**Comment by rfc1036** [comment]
Do you have a specific reason to use Net::Whois::RIPE? Nowadays in most cases you can just use the bgpq3 backend, which is much faster.

**Comment by netravnen** [comment]
Net::Whois::RIPE is because it is stated in the requirements (./INSTALL) for using rpsltool.

**Comment by netravnen** [comment]
> Nowadays in most cases you can just use the bgpq3 backend

You mean to use bgpq3 instead of rpsltool?

**Comment by rfc1036** [comment]
No, I mean using bgpq3 as a backend for rpsltool. Net::Whois::RIPE is a requiremente only if you want to use the Net::Whois::RIPE backend.

**Comment by netravnen** [comment]
Is there a way to tell rpsltool to ingore Net::Whois::RIPE not being installed? (And use the bgpq3 backend instead)

Currently I get this:
```
/opt/rpsltool > ./rpsltool
Can't locate Net/Whois/RIPE.pm in @INC (you may need to install the Net::Whois::RIPE module) (@INC contains: /opt/rpsltool/lib /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.24.1 /usr/local/share/perl/5.24.1 /usr/lib/x86_64-linux.
BEGIN failed--compilation aborted at /opt/rpsltool/lib/RPSLToolWhois.pm line 4.
Compilation failed in require at ./rpsltool line 20.
BEGIN failed--compilation aborted at ./rpsltool line 20.
```
When trying to run rpsltool from the install dir.
